Adyen/UI/Form/Items/SelectableFormItem/SelectableFormItemView.swift

```diff
@@ -0,0 +1,180 @@
+//
+// Copyright (c) 2024 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import UIKit
+
+/// A view representing a selectableFormItem item.
+@_spi(AdyenInternal)
+public final class SelectableFormItemView: FormItemView<SelectableFormItem> {
+    private let imageLoader: ImageLoading = ImageLoaderProvider.imageLoader()
+    private var imageLoadingTask: AdyenCancellable?
+
+    // MARK: - ImageView
+
+    private lazy var imageView: UIImageView = {
+        let imageView = UIImageView(image: UIImage(named: "upiLogo",
```

Comment:
Let's move image names and width/height values to an private constant enum

Reply:
8079ceeba101a6e70258f18631cbfa9367c7b89e